.travis.yml

```diff
@@ -1,2 +1,2 @@
 language: node_js
-node_js: '0.10'
+node_js: '4'
```

Comment:
please keep 0.10 here - also add 0.12, iojs, 6, 8, and 9.

Reply:
If I keep at `0.10` the tests wont run as it's using syntax only available in `4` and above.

Also eslint only supports `4` and above as far as I know. There may be a much only version that supports `0.10`. Will take a look a bit later.

Reply:
Okay, `eslint@2.13` supports `0.10`: https://github.com/eslint/eslint/blob/031a35614f9353b57c072ba14aff16f930ab6520/package.json

Reply:
The better solution is to not run eslint on every node version, since that doesn't give you any useful information. eslint should only be running on a single job; the matrix shouldn't be running the linter.

Reply:
@ljharb never configured Travis for something like that before. Is this correct, or is there a nicer way? https://stackoverflow.com/questions/28652664/specifying-node-version-for-a-certain-test

Reply:
https://github.com/ljharb/is-callable/blob/master/.travis.yml may be helpful

Reply:
OOoh, `TRAVIS_NODE_VERSION`, nice.